templates/admin/user/edit.tmpl

```diff
@@ -189,7 +189,7 @@
 
 				<div class="field">
 					<button class="ui green button">{{$.locale.Tr "settings.update_avatar"}}</button>
-					<a class="ui red button delete-post" data-request-url="{{.Link}}/avatar/delete" data-done-url="{{.Link}}">{{$.locale.Tr "settings.delete_current_avatar"}}</a>
+					<a class="ui red button delete-post" data-request-url="{{.Link}}/avatar/delete" data-done-url="{{.Link}}">{{$.locale.Tr "settings.delete_current_avatar"}}</a>{{/* TODO: Why is this link not focusable? */}}
```

Comment:
Because it doesn't have a `href`.

Reply:
Ah, sounds like a problem for another PR.
I've just had a look, this means there are roughly as many unfocusable links as there were pseudo-buttons before this PR.

Reply:
I believe there was no strong reason. In old code, the AJAX buttons are mostly written as `<a>`, but I think it's more correct to write them as `<button>` now, otherwise there is still a a11y problem .....